packages/bridge-ui-v2/src/components/Bridge/Bridge.svelte

```diff
@@ -142,6 +147,9 @@
   }
 
   async function bridge() {
+    isBridgePaused().then((paused) => {
+      if (paused) throw new BridgePausedError('Bridge is paused');
+    });
     if (!$bridgeService || !$selectedToken || !$network || !$destinationChain || !$account?.address) return;
```

Comment:
Recommended, optional.

`if (notAllowBridge()) return;`
Create function to check allowBridge
 
```
 function notAllowBridge() {
  const conditions = [
    !$bridgeService,
    !$selectedToken,
    !$network,
    !$destinationChain,
    !$account?.address
  ];

  return conditions.some(Boolean);
}
```